src/index.css

```
@@ -82,10 +100,10 @@
   --color-danger: var(--c-danger);
   --color-danger-soft: var(--c-danger-soft);
 
-  /* Inter for UI, Zen Maru Gothic for headwords, Noto Sans TC for CJK prose. */
-  --font-sans: 'Inter', system-ui, sans-serif;
+  /* System UI for chrome, self-hosted Zen Maru Gothic for headwords, platform CJK faces for prose. */
+  --font-sans: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)